docs/app/sidebars.js

```diff
@@ -91,6 +82,9 @@ const sidebars = {
             type: 'doc',
             id: 'configuration'
         }, {
+            type: 'doc',
+            id: 'devbox_cloud/beta_faq'
```

Comment:
devbox_playground?

Reply:
See above, does changing the URL structure cause any concerns?

Reply:
You are talking about incoming links right? I think it's fine. Most incoming links go to devbox.sh anyway